content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -353,7 +352,11 @@ In the CLI, the access modes are abbreviated to:
 * ROX - ReadOnlyMany
 * RWX - ReadWriteMany
 
-> __Important!__ A volume can only be mounted using one access mode at a time, even if it supports many.  For example, a GCEPersistentDisk can be mounted as ReadWriteOnce by a single node or ReadOnlyMany by many nodes, but not at the same time.
+{{< caution >}}
+
+A volume can only be mounted using one access mode at a time, even if it supports many.  For example, a GCEPersistentDisk can be mounted as ReadWriteOnce by a single node or ReadOnlyMany by many nodes, but not at the same time. Besides, it will be limited once a volume has been mounted on a single node with `ReadWriteOnce`.
```

Comment:
> Besides, it will be limited once a volume has been mounted on a single node with `ReadWriteOnce`.

This sentence is hard to parse.

Reply:
Do you have suggestion about this?

Reply:
I would suggest drop the last sentence "Besides, it will be limited once a volume has been mounted on a single node with `ReadWriteOnce`."

Reply:
@Colstuwjx , The `caution` shortcode is a great addition. Would you remove the last sentence?
I am not sure if it is clear what is `limited`.